composer.json

```diff
@@ -1,13 +1,15 @@
 {
     "name": "davidpiesse/nova-toggle",
-    "description": "A Boolean Toggle Switch Field",
+    "description": "A Laravel Nova Toggle field.",
     "keywords": [
         "laravel",
-        "nova"
+        "nova",
+        "toggle",
+        "field"
     ],
     "license": "MIT",
     "require": {
-        "php": ">=7.1.0"
+        "php": "^7.3|^8.0"
```

Comment:
please add "laravel/nova": "^4.0"

Reply:
This should not be included in the package as I understand it as it forces when developing the package on its own to require a license key be used (included somewhere).

Reply:
nothing to do with license. just a requirement that this package can only be use on nova 4 above